pettingzoo/butterfly/knights_archers_zombies/manual_policy.py

```diff
@@ -0,0 +1,77 @@
+import pygame
+
+
+class ManualPolicy:
+    def __init__(self, env, agent_id: int = 0, show_obs: bool = False):
+
+        self.env = env
+        self.agent_id = agent_id
+        self.agent = self.env.agents[self.agent_id]
+
+        # TO-DO: show current agent observation if this is True
+        self.show_obs = show_obs
+
+        # action mappings for all agents are the same
+        if True:
+            self.default_action = 5
+            self.action_mapping = dict()
+            self.action_mapping[pygame.K_w] = 0  # front
+            self.action_mapping[pygame.K_s] = 1  # back
+            self.action_mapping[pygame.K_q] = 2  # rotate left
+            self.action_mapping[pygame.K_e] = 3  # rotate right
+            self.action_mapping[pygame.K_f] = 4  # weapon
```

Comment:
Is wsqe a known key mapping? I found it a bit unfamiliar. But I could just be out of the loop.

Reply:
Originally I used WASD and space for attack, but then manual_control.py uses WSQE and F, and uses AD to change agents. I wasn't sure what tonuse, so stuck with the one in manual_control.py. That said, this can be easily changed, though I suspect similae issues will crop up in other envs.

Reply:
Could you please use WASD and space for attack and then use QE to switch agents? I think the keybindings were done by Mario years ago with zero review